cmake/modules/luajit.cmake

```diff
@@ -77,6 +77,9 @@ else()
 					UPDATE_COMMAND ""
 					INSTALL_COMMAND "")
 			endif()
+			install(FILES "${LUAJIT_LIB}" DESTINATION "${CMAKE_INSTALL_LIBDIR}/${LIBS_PACKAGE_NAME}/")
+			install(DIRECTORY "${LUAJIT_INCLUDE}/" DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/${LIBS_PACKAGE_NAME}/"
```

Comment:
Why not using same fix as other cmake modules here, ie: adding the trailing slash to `set(LUAJIT_INCLUDE ...` line, for coherence?